### PR TITLE
first pass at an svg component for the MOTI letters

### PR DESCRIPTION
**Pull request by dhunninghake**

### CODE REVIEW

**Comment by hhff** [comment]
woah dis is sick

File: app/components/Icon.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React, { Component, PropTypes } from 'react';
+
+export default class Icon extends React.Component {
```
Comment (hhff) [inline app/components/Icon.js]: We dont need to do React.Component, we can just do Component

File: app/components/Icon.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+Icon.propTypes = {
+  icon: React.PropTypes.string.isRequired,
+  viewbox: React.PropTypes.string.isRequired,
+  width: React.PropTypes.number,
+  height: React.PropTypes.number,
+  classes: React.PropTypes.object,
+  fill: React.PropTypes.string
+}
```
Comment (hhff) [inline app/components/Icon.js:27]: more a style thing, but I'd opt for putting propTypes inside the component, at the top of the object above the contstructor.  This is super useful information I don't want other devs skipping / missing

Comment (hhff) [inline app/components/Icon.js:27]: we also deconstructed proptypes, so you don't need to write React each time

**Comment by dhunninghake** [comment]
Sweet! Those issues are fixed now 🙌🏼

**Comment by sepowitz** [comment]
LGTM!

**Pull request merged**